frontend/src/redux/exams/reducers.js

```diff
@@ -29,13 +36,31 @@ export default combineReducers({
   loading: (state = false, action) => {
     switch(action.type) {
       case ACTIONS.SET_GET_EXAMS_START:
+      case ACTIONS.SET_GET_EXAM_START:
+      case ACTIONS.SET_POST_EXAM_START:
         return true;
       case ACTIONS.SET_GET_EXAMS_SUCCESS:
+      case ACTIONS.SET_GET_EXAM_SUCCESS:
+      case ACTIONS.SET_POST_EXAM_SUCCESS:
       case ACTIONS.SET_GET_EXAMS_FAILURE:
+      case ACTIONS.SET_GET_EXAM_FAILURE:
+      case ACTIONS.SET_POST_EXAM_FAILURE:
         return false;
 
       default:
         return state;
     }
   },
+
+  errorMessage: (state = null, action) => {
+    const { type, err } = action;
+
+    if (type === ACTIONS.RESET_ERROR_MESSAGE) {
```

Comment:
is it a pattern that you reproduced ?
(I have no idea on how to handle errors with redux, sorry I'm a noob)

Reply:
Yes, commented on the code!